lxd/util/sys.go

```diff
@@ -79,3 +89,70 @@ func GetIdmapSet() *idmap.IdmapSet {
 	}
 	return idmapSet
 }
+
+func RuntimeLiblxcVersionAtLeast(major int, minor int, micro int) bool {
+	version := C.GoString(C.lxc_get_version())
```

Comment:
What's the reason for going directly through lxccontainer.h rather than use go-lxc's Version() function?

Looks like doing: strings.Replace(lxc.Version(), " (devel)", "-devel") would get you the same string that lxc_get_version() gives you.

Reply:
I totally forgot that we export that function in go-lxc already because I only looked at `container.go` not `lxc-container.go` too. The funny part is that I wrote most of the golxc `Version()` function...